packages/template-retail-react-app/app/hooks/use-auth-modal.js

```diff
@@ -317,14 +321,17 @@ export const AuthModal = ({
                     {form.formState.isSubmitSuccessful && currentView === PASSWORD_VIEW && (
                         <PasswordResetSuccess />
                     )}
+                    {!form.formState.isSubmitSuccessful && currentView === EMAIL_VIEW && (
+                        <CheckEmail email={submittedEmail.current} />
+                    )}
```

Comment:
not sure if this is where the validation should be but i'm finding that I'm able to submit the auth-modal with an empty email but not from the /login page
<img width="332" alt="Screenshot 2024-11-06 at 2 46 51 PM" src="https://github.com/user-attachments/assets/8d1d0f30-c46e-4b64-9e2c-0537c57d0ad4">

Reply:
@hajinsuha1 Did some refactoring - users should **NOT** be able to click the **Passwordless Login** button without inputting an email in the **Email** field first!